Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -1768,16 +2018,14 @@ module.exports = {
       .addExpression(
         'Inertia',
         _('Inertia'),
-        _(
-          'Return the rotational inertia of the object (in kilograms * meters * meters)'
-        ),
+        _('Return the rotational inertia of the object (kg · pixels²)'),
```

Comment:
This is a breaking change but:
- I doubt it's used much.
- It can't be used in extensions because there isn't any expression to get the world scale.